crmsh/bootstrap.py

```diff
@@ -1317,8 +1322,6 @@ def init_admin():
             return
 
         adminaddr = prompt_for_string('Virtual IP', valid_func=Validation.valid_admin_ip)
-        if not adminaddr:
-            utils.fatal("Expected an IP address")
```

Comment:
This 2 lines are useless now, since `prompt_for_string` will ask user in loop until get the input, or, this value will be passed by -A option